### PR TITLE
Milestone 05: Refactor Code

### PR DESCRIPTION
**Pull request by FaranoshAmini**
### In this Pull request, We have implemented the following:

  - Used correctly [Gitflow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md)
  - Documented our work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md)
  - Paid special attention to the "Single-responsibility Principle".
- Made sure that the role of `main.rb` is only to start the app and ask the user for the initial choice of options (or to quit).
- Extracted some pieces of logic into separate class(es).

### CODE REVIEW

File: classes/app.rb [hunk 1]
```diff
@@ -1,164 +1,112 @@
...
+  #   puts '1 - Create Student  2 - Create Teacher'
+  #   select_option
+  #   person = gets.chomp
+  #   print 'Insert age: '
+  #   age = gets.chomp
+  #   print 'Insert name: '
+  #   name = gets.chomp
+  #   person_condition(person, age, name)
+  #   puts 'Person Created Successfully'
+  #   back_to_menu
```
Comment (uwadonat) [inline classes/app.rb]: **_Optional:_**
- [ ] Please remove all these commented lines to make your project more clean.

**Review by uwadonat — APPROVED** [review]
## Hi @Team,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### Congratulations! 🎉 
### Highlights
- Professional pull request message ✔️
- No linter errors ✔️
- Well organized project ✔️

# Status: Approved ✔️✔️✔️

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100%.
Please, remember to tag me @uwadonat in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: classes/app.rb (hunk 1 above)
Comment (FaranoshAmini) [reply classes/app.rb]: I have removed the comments, Thank you

**Comment by FaranoshAmini** [comment]
@uwadonat Thank you for your kind review and feedback.

**Pull request merged**